### ISSUE TITLE
Access is Denied Error on Updating Mod

### ISSUE THREAD

**Issue by firefly2442** [headswe/Swifty#110]
I get the following error upon updating the "ALIVE" mod.

"Error!  Access is Denied" in red.

```
2015-12-22 19:43:49,397 [6] INFO SwiftUpdater.Pages.Download - Starting download for demo\sp\alive_divide_and_rule.altis\readme.txt_0.spd
2015-12-22 19:43:49,421 [6] INFO SwiftUpdater.Pages.Download - Starting download for demo\sp\basic.stratis\readme.txt_0.spd
2015-12-22 19:43:50,776 [6] INFO SwiftUpdater.Pages.Download - Crash
System.ComponentModel.Win32Exception (0x80004005): Access is denied
   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.NtProcessManager.GetModuleInfos(Int32 processId, Boolean firstModuleOnly)
   at System.Diagnostics.NtProcessManager.GetFirstModuleInfo(Int32 processId)
   at System.Diagnostics.Process.get_MainModule()
   at SwiftUpdater.TeamspeakUtilities.CheckTeamspeakPlugins(Repository repo) in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\TeamspeakUtilities.cs:line 33
   at SwiftUpdater.Pages.Download.Worker_DoWork(Object sender, DoWorkEventArgs e) in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\Pages\Download.cs:line 323
2015-12-22 19:43:50,780 [1] INFO SwiftUpdater.Pages.RepoView - Updateing view
```

I tried deleting the entire folder and having Swifty download the entire mod but that also didn't work.

**Comment by ryanparsons7** [comment]
Confirmed, also getting this when updating @task_force_radio

**Comment by headswe** [comment]
Its trying to update teamspeak folders, but swifty does not run in forced admin mode anymore.

**Comment by firefly2442** [comment]
I tried it with the option "autocopy teamspeak plugins" off and I don't get an error message but it doesn't seem to finish the synchronization.  If I click update again, there's still the same update for the ALIVE mod.  I also noticed when Teamspeak is running, the error message comes up.  However, the autocopy option is off which doesn't make sense.  Perhaps there is some additional check or attempt at copying that's going on even when this setting is off?
